test/__init__.py

```diff
@@ -11,4 +11,11 @@ def tests():
     suite = unittest.TestSuite()
     suite.addTest(unittest.makeSuite(rlite.RliteTest))
     suite.addTest(unittest.makeSuite(persistent.PersistentTest))
+
+    try:
+        from test import patch
+        suite.addTest(unittest.makeSuite(patch.PatchConnTest))
```

Comment:
I don't think the tests you added run on Travis[1], I believe redis package is not installed... it'd be nice to have a build matrix so we can test with and without redis[2]

[1] https://travis-ci.org/seppo0010/rlite-py/jobs/186281584

[2] https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix